splitgraph/ingestion/sqlite/__init__.py

```diff
@@ -35,6 +36,8 @@
 ORDER BY 1,2;
 """
 
+RE_SINGLE_PARAM_TYPE = re.compile(r"^([A-Z]+)\(([0-9]+)\)$")
+RE_DOUBLE_PARAM_TYPE = re.compile(r"^([A-Z]+)\(([0-9]+),([0-9]+)\)$")
```

Comment:
Could insert some optional whitespace matchers between e.g. `VARCHAR` and `(` in case there's some weirdness going on with the type names that sqlite gives us.